MojioSDK/Models/Core/Media.swift

```diff
@@ -0,0 +1,93 @@
+/******************************************************************************
+ * Moj.io Inc. CONFIDENTIAL
+ * 2017 Copyright Moj.io Inc.
```

Comment:
I think we should update Copyright in SDK

Reply:
especially new files